pkg/broker/broker.go

```diff
@@ -105,18 +107,19 @@ func (b *Broker) Provision(request *osb.ProvisionRequest, _ *broker.RequestConte
 		response.OperationKey = &operationKey
 	}
 
-	glog.V(5).Infof("Successfully initiated provisioning %s (%s/%s) in %s", request.InstanceID, request.ServiceID, request.PlanID, namespace)
+	klog.V(4).Infof("broker: provisioned %q in namespace %q", request.InstanceID, namespace)
```

Comment:
👍 being a trace-level thing, I can see this in 4 or 5.

Reply:
I'll leave in 4.